### ISSUE TITLE
[Issue]: Allow remote connections checkbox missing

### ISSUE THREAD

**Issue by quielb** [jellyfin/jellyfin#8063]
### Please describe your bug

When trying to modify a user in the admin dashboard, there is no checkbox for "Allow remote connections to this Jellyfin Server" for any of the users. Also checked when creating a new user that checkbox is missing. At some point I created a user without remote access and now I can't change that property via Chrome or Android app because the checkbox is missing. When creating a new user the checkbox is also not present, but it seems the default is to allow remote access.  Short term workaround would be to delete user and re-create accepting default of allowing remote access. Only problem is when deleting user, watch history is also lost.

### Jellyfin Version

10.8.0

### Environment

```markdown
- Virtualization: Docker using jellyfin container
- Clients: Chrome and Android app
```

### Jellyfin logs

```shell
Jellyfin.Server.Implementations.Users.UserManager: Authentication request for "erik" forbidden: remote access disabled and user not in local network (IP: "172.58.39.203").
```

### Please attach any screenshots here

![image](https://user-images.githubusercontent.com/13806281/173263521-a3d8a834-95eb-4e3f-944f-64ba15556bce.png)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by thornbill** [comment]
Transferring back to the server repo since this was caused by a change to the api. 🙃

**Comment by anthonylavado** [comment]
Looks like this is fixed as of 10.8.2.
